files_daily.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+for SITE in `ls -d docroot/sites/*/`; do
+  SITE=`basename $SITE`
+
+  if [[ $SITE == 'all' ]] || [[ $SITE == 'default' ]]; then
```

Comment:
What if a site is not a multisite and we'd like to backup the default folder?

Reply:
that's true. I updated it to not skip default if not a multisite.